vcpkg.json

```diff
@@ -19,6 +19,7 @@
     "qt5-location",
     "qt5-charts",
     "qt5-multimedia",
+    "qt5-serialport",
```

Comment:
```suggestion
    {
      "name": "qtserialport",
      "platform": "windows"
    },
```

Reply:
I thought it's "only" a BT on Win hack -- but you convinced me already on backchannels